otj-metrics-core/src/main/java/com/opentable/metrics/graphite/GraphiteConfiguration.java

```diff
@@ -206,4 +215,15 @@ public static ClusterNameType fromParameterName(String parameterName) {
                     .findFirst().orElse(ClusterNameType.PART_OF_INSTANCE);
         }
     }
+
+    private Optional<HostAndPort> getHostPort() {
+        if (StringUtils.isNotBlank(graphiteHost)) {
+            return Optional.of(HostAndPort.fromParts(graphiteHost, graphitePort));
+        }
+        if (StringUtils.isNotBlank(twelveFactorURL)) {
```

Comment:
config takes precedence over 12 factor?

Reply:
Yes, I think if they explicitly used the old system to customize it, it should continue to work IMO. Of course this has negatives so I'm open to discussion @adamzr